ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -269,7 +269,7 @@ def __init__(
                 self._itr_skipping_level,
                 self._git_data,
                 self._configurations,
-                self.tracer._agent_url,
+                agent_config.trace_agent_url if not self.tracer._agent_url else self.tracer._agent_url,
```

Comment:
Can we use an `or` here like the one below?
```suggestion
                self.tracer._agent_url or agent_config.trace_agent_url,
```